src/main.ts

```diff
@@ -0,0 +1,67 @@
+import { Command, Helper } from 'dojo-cli/interfaces';
+import { Argv } from 'yargs';
+import runTests from './runTests';
+
+export interface TestArgs extends Argv {
+	environments: string;
+	config: string;
+	unit: boolean;
+	functional: boolean;
+	reporters: string;
+	coverage: boolean;
+}
+
+const command: Command = {
+	description: 'test your application',
+	register(helper: Helper) {
+		helper.yargs.option('c', {
+			alias: 'config',
+			describe: 'Specifies what configuration to test with: browserstack(default), \'saucelabs\', or \'local\'ly.',
```

Comment:
could also be testingbot or crossbrowsertesting (though perhaps not since we don't include configs for those environments, but perhaps we should)?